src/System.IO.FileSystem.Watcher/ref/project.json

```diff
@@ -1,7 +1,8 @@
 {
   "dependencies": {
     "System.Runtime": "4.4.0-beta-24714-01",
-    "System.ComponentModel.Primitives": "4.4.0-beta-24714-01"
+    "System.ComponentModel.Primitives": "4.4.0-beta-24714-01",
+    "System.ComponentModel.TypeConverter": "4.4.0-beta-24714-01"
```

Comment:
>4.4.0-beta-24714-01 [](start = 44, length = 19)

Just as FYI, whenever adding dependencies like in here, make sure that you rebase and update the dependency version before merging, since it is likely that they will change from the moment you sent out your review to the moment you merge.